AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -393,6 +400,7 @@ private void AddErrorsToModelState(IdentityResult result)
             }
         }
 
+        //TODO: either extract this into a provider pattern or somehow decide which of these conditionals are invoked by which caller and get rid of this method.

```

Comment:
There is the new `LocalRedirect` controller base method that could be considered.